src/sources/statsd/parser.rs

```diff
@@ -105,6 +112,21 @@ fn sanitize_key(key: &str) -> String {
     s.into()
 }
 
+fn sanitize_sampling(sampling: f32) -> f32 {
+    if sampling == 0.0 {
+        1.0
+    } else {
+        sampling
+    }
+}
+
+fn convert_to_base_units(unit: &str, val: f32) -> f32 {
+    match unit {
+        "ms" => val / 1000.0,
```

Comment:
What is our base unit? If we're taking in milliseconds and dividing by 1000, is it microseconds? It seems weird to assume incoming stats with the `h` tag are in microseconds.

Reply:
I assumed the base unit is seconds.

Reply:
This conversion came to my mind when I was browsing [this](https://github.com/prometheus/statsd_exporter/blob/master/exporter.go#L216) code for statsd to prom exporter.

Reply:
Oh wow, too early for my brain to do math apparently. That makes more sense.

I'm not totally sure we want seconds as our standard base unit, but we can go with this for now since it matches prometheus and reevaluate as we add more sources and sinks.